daemon/daemon.go

```diff
@@ -29,12 +29,14 @@ func setUpHealthCheckService(port string, endpoint string) {
 	mux := http.NewServeMux()
 	mux.HandleFunc(endpoint, func(rw http.ResponseWriter, req *http.Request) {
 		rw.WriteHeader(http.StatusOK)
+		log.Printf("Health-check endpoint pinged.")
```

Comment:
this is the important one that should be a counter going through kayvee (the other ones are fine as is)